README.md

```diff
@@ -1,217 +1,212 @@
+   1. [Initial Play Store upload](#initial-play-store-upload)
+   1. [Service Account](#service-account)
+1. [Usage](#usage)
+   1. [Installation](#installation)
+   1. [Sample tasks](#sample-tasks)
+   1. [Authenticating the plugin](#authenticating-the-plugin)
+1. [Configuration](#configuration)
+   1. [Specify the track](#specify-the-track)
+   1. [Untrack conflicting versions](#untrack-conflicting-versions)
+   1. [Play Store metadata](#play-store-metadata)
+   1. [Uploading images](#uploading-images)
+   1. [Using multiple Service Accounts](#using-multiple-service-accounts)
+   1. [Running custom tasks before publishing](#running-custom-tasks-before-publishing)
+
+## Quick start guide
+
+1. Upload the first version of your APK or App Bundle using the
+   [Google Play Console](https://play.google.com/apps/publish)
+1. [Create a Google Play Service Account](#google-play-service-account)
+1. [Sign your release builds](https://developer.android.com/studio/publish/app-signing#gradle-sign)
+   with a valid `signingConfig`
+1. [Add and apply the plugin](#installation)
+1. [Authenticate the plugin](#authenticating-the-plugin)
 
 ## Prerequisites
 
-### Initial Play Store Upload
-
-The first APK of your App needs to be uploaded via the web interface. This is to register the application id and cannot be done using the Play Developer API. For all subsequent uploads and changes this plugin can be used.
+### Initial Play Store upload
 
-### Google Play Service Account
+The first APK or App Bundle needs to be uploaded via the Google Play Console because registering the
+app with the Play Store cannot be done using the Play Developer API. For all subsequent uploads and
+changes this plugin can be used.
 
-To use the publisher plugin you have to create a service account for your existing Google Play Account. See https://developers.google.com/android-publisher/getting_started for more information.
+### Service Account
 
-Due to the way the Google Play Publisher API works, you have to grant at least the following permissions to that service account:
+To use this plugin, you must create a service account with access to the Play Developer API. You'll
+find a guide to setting up the service account
+[here](https://developers.google.com/android-publisher/getting_started#using_a_service_account).
+Once that's done, you'll need to grant the following permissions to your service account for this
+plugin to work (go to Settings -> Developer account -> API access -> Service Accounts):
 
 ![permissions.png](https://cloud.githubusercontent.com/assets/242983/17809992/95ea4eaa-661a-11e6-9879-521df4f14735.png)
 
-Once you finished the setup you have a so called *service account email address* and a *p12 key file* that we will use later on.
-
-### Signing Configuration
-
-Please make sure to assign a valid signing configuration to your release build type. Otherwise, there won't be a publishable (signed) APK. In that case, the plugin won't create any of its tasks.
```

Comment:
I'd like to keep a hint for that as long as we're not changing the behavior. This has been a constant source of error when setting up the plugin

Reply:
You're right. In the future, it'd be nice to just break the build with a pretty error message, see #268.